<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaCalendario</class>
 <widget class="QDialog" name="VentanaCalendario">
  <property name="windowTitle">
   <string>Calendario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text"><string>Calendario</string></property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listPartidos"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hButtons">
     <item>
      <widget class="QPushButton" name="btnPlayNext">
       <property name="text"><string>Jugar siguiente</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAutoGenerate">
       <property name="text"><string>Autogenerar resultado</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCerrar">
       <property name="text"><string>Cerrar</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
